app/bundles/UserBundle/Assets/js/user.js

```diff
@@ -9,6 +9,12 @@ Mautic.userOnLoad = function (container) {
             Mautic.activateSearchAutocomplete('list-search', 'user.user');
         }
     }
+    if(mQuery("#user_automaticPassword_1").prop('checked')) {
+        const randomPassword = Math.random().toString(36).slice(-8);
+        mQuery("#user_plainPassword #user_plainPassword_password").val(randomPassword);
+        mQuery("#user_plainPassword #user_plainPassword_confirm").val(randomPassword);
+        mQuery("#user_plainPassword").toggleClass("hide");
+    }
```

Comment:
This looks like duplication of the new `Mautic.togglePasswordInput()` function. Please reuse that code.